src/www/ui/api/Controllers/UploadController.php

```diff
@@ -94,11 +96,30 @@ class UploadController extends RestController
    */
   const VALID_STATUS = ["open", "inprogress", "closed", "rejected"];
 
+  /**
+   * Agent list
+   */
+  private $agentNames = AgentRef::AGENT_LIST;
```

Comment:
Would suggest to use `AgentRef::AGENT_LIST` directly wherever needed to make it easy to identify in future.